pkg/collector/corechecks/cluster/kubernetes_state.go

```diff
@@ -338,7 +340,7 @@ func (k *KSMCheck) buildTag(key, value string) (tag, hostname string) {
 	var sb strings.Builder
 	sb.Grow(len(key) + 1 + len(value))
 	sb.WriteString(key)
-	sb.WriteRune(':')
+	sb.WriteByte(':')
```

Comment:
Curious - is it faster in the end? 😄

Reply:
My bench it too imprecise to measure it. But by looking at the code, it should. ;-)